znoyder/config.yml

```diff
@@ -133,13 +133,6 @@ add:
           - 'check'
           - 'gate'
 
-  'keystone':
-    'osp-17.0':
-      'openstack-tox-py39':
-        type:
-          - 'check'
-          - 'gate'
-
```

Comment:
Please do not remove this – openstack-tox-py39 is not defined in upstream for Keystone (other projects have it as non-voting by default), so this is what this entry exists in add map.

Reply:
Right, I'll add it back, thanks!

Reply:
Thanks, now it is great :-) :+1: